packages/plugin-typescript/package.json

```diff
@@ -5,7 +5,9 @@
   "dependencies": {
     "@berry/builder": "workspace:*",
     "@berry/core": "workspace:*",
-    "@berry/plugin-essentials": "workspace:*"
+    "@berry/plugin-essentials": "workspace:*",
+    "@types/inquirer": "6.0.1",
```

Comment:
Should the types be here? 🤔

Reply:
Types probably should be declared at the place the `typeRoots` configuration is set. At the moment it's at the root of the repository.

@vlasenko's work will likely remove the need for `typesRoot` in the medium term, which would make it possible to store the types in each individual package.